cg/services/analysis_starter/configurator/file_creators/params/raredisease.py

```diff
@@ -0,0 +1,100 @@
+from pathlib import Path
+
+from cg.apps.lims import LimsAPI
+from cg.constants import DEFAULT_CAPTURE_KIT
+from cg.constants.nf_analysis import NextflowFileType
+from cg.constants.scout import ScoutExportFileName
+from cg.constants.tb import AnalysisType
+from cg.exc import CgDataError
+from cg.io.yaml import read_yaml
+from cg.models.raredisease.raredisease import RarediseaseParameters
+from cg.services.analysis_starter.configurator.file_creators.abstract import FileContentCreator
+from cg.services.analysis_starter.configurator.file_creators.params.utils import (
+    replace_values_in_params_file,
+)
+from cg.services.analysis_starter.configurator.file_creators.utils import (
+    get_case_id_from_path,
+    get_file_path,
+)
+from cg.store.models import BedVersion, Case, Sample
+from cg.store.store import Store
+
+
```

Comment:
The params file needs an implementation per pipeline because each one uses a different pydantic model to parse parameters